### DISCUSSION TITLE
Error: Cannot trigger a 'workflow_dispatch' on a disabled workflow

### DISCUSSION THREAD

**Discussion by peteryang1756** [upptime/upptime#593] [Q&A] [answered]
How to solve:Error: Cannot trigger a 'workflow_dispatch' on a disabled workflow

**Comment by AnandChowdhary** [comment, marked as answer]
I've never seen this before, but you should enable a disabled workflow first, and then trigger the event.

**Reply by peteryang1756** [reply]
Thank you!
